server/utils/auth.js

```diff
@@ -0,0 +1,45 @@
+const jwt = require('jsonwebtoken');
+
+const secret = 'mysecretsTokenStringss';
```

Comment:
could we add this to .env file instead? I'll make sure to also add it to prod env vars set on heroku